### PR TITLE
Accept rho, theta in extrapolate_impurity_density.

### LINKED ISSUE #150: Allow extrapolate impurity density taking rho, theta rather than R, z
`ExtrapolateImpurityDensity` currently only takes `impurity_density_sxr` on R, z. Would be useful if it could operate taking rho, theta.

### PR DESCRIPTION
**Pull request by k-collie**
Fixes #150 

Needs more work to verify asymmetry_from_rho_theta.
Currently returning centre full on NaNs.

### CODE REVIEW

**Comment by k-collie** [comment]
`extrapolator.asymmetry_modifier` has nans in same places as returned density.

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ indica/git_version
 *.png
 *.prof
 *.lprof
+.ropeproject/
```
Comment (gadgil48) [inline .gitignore:19]: Was this a temporary change or is it still necessary?

Comment (k-collie) [reply .gitignore:19]: They're files generated by a refactoring tool which shouldn't be committed afaict so I don't think it's necessary but doesn't hurt either.

**Review by gadgil48 — APPROVED** [review]
Looks good, if possible some more tests to satisfy `codecov` would be nice (just for the sake of green tick OCD) but if it's not possible/not useful then it's fine (we can always reset `codecov` before beta release anyway)

**Pull request merged**